### ISSUE TITLE
Multiple Notebooks

### ISSUE THREAD

**Issue by ghost** [janwillemb/Ema-Personal-Wiki#2]
It would be nice if it were possible to create multiple notebooks -- I currently have a ton of pages that are grouped for various topics (i.e. work, personal projects, random ideas, etc.).  While I can organize these under separate "home" pages, it'd be cleaner and easier if I could put them into completely different notebooks altogether.

Another benefit is that it'd reduce the potential for page name clashes.  For example, I might have a personal programming project and want a page titled DatabaseStructureIdeas; that could easily clash with a similar page for a work project -- or even another personal project.

Not completely sure what would be a reasonable file-based structure for this; I was thinking sub-directories might work, but that might clash with another issue on here.  However, that'd also allow for having nested notebooks/groups of notebooks fairly easily, if that were desired.

**Comment by matmatmatmatmatmat** [comment]
I agree this feature would be great, I have the same crowded home page problem. Maybe installing the app as many times as the number of homepage needed would work ? I'm trying to re compile another version of the app ("trying" because I just started learning android development)
